CSS-10880 Improve documentation (#53)
canonical/superset-k8s-operator: documentation/tutorial/01-introduction.md

> # Charmed Superset

Reviewer: ```suggestion
# Get started with Charmed Superset
```

This guide describes how you can get started with Charmed Superset. From setting up MicroK8s in your host environment, all the way to creating your first dashboard.

> Charmed Superset delivers automated operations management from day 0 to day n for [Apche Superset](https://superset.apache.org/). It is an open source, end-to-end, production-ready data visualization tool on top of [Juju](https://juju.is/).

Reviewer: ```suggestion
Charmed Superset delivers automated operations management from day zero to day n for [Apache Superset](https://superset.apache.org/). It is an open source, end-to-end, production-ready data visualization tool on top of [Juju](https://juju.is/).
```

## Prerequisites

- Ubuntu 22.04 LTS or later.
- [Snap](https://snapcraft.io/) installed.
- 8GB of RAM.
- 2 CPU threads.

> - At least 20GB of available storage.

Reviewer: ```suggestion
- At least 20GB of available disk space.
```

> - Access to the internet for downloading the required rocks and charms.

Reviewer: Is this one needed? It seems to be obvious nowadays, had the same question in other Canonical products

Author: Probably not, it's unlikely anyone will deploy superset to an air-gapped environment. I'll remove it.

## Get Started

> - [Environment Setup]()

Reviewer: It would better for UX to have numbered bullet points.
Also, I am fine with breaking the tutorial into several pages, but based on the content I see, everything one page would be also OK. Up to you.

Reviewer: Titles need to be updated. You will find my suggestions in the dedicated pages below.
